Online/OnlineManager.cs

```diff
@@ -35,16 +35,16 @@ public OnlineManager(ProcessManager manager) : base(manager, RainMeadow.Ext_Proc
             LeaveLobby();
             MatchmakingManager.instance.OnLobbyJoined += OnlineManager_OnLobbyJoined;
             RainMeadow.Debug("OnlineManager Created");
-
         }
 
-
         private void OnlineManager_OnLobbyJoined(bool ok, string error)
         {
             RainMeadow.Debug(ok);
             currentlyJoiningLobby = default;
             if (ok)
             {
+                manager.rainWorld.progression.Destroy();
```

Comment:
putting these here because i wouldn't expect gamemodes to do it themselves manually